### ISSUE TITLE
It doesn't work with gatsby build --prefix-paths

### ISSUE THREAD

**Issue by isi-gach** [wardpeet/gatsby-plugin-static-site#13]
I have created an example repo to reproduce the issue:
https://github.com/isi-gach/gastby-static/tree/create-react-app

When I do `npm run build:gatsby` and I go to http://localhost:5000/demo/index.html
the browser URL changes

If I create a `create-react-app` build instead using `npm run build` and I go to http://localhost:5000/demo/index.html the URL doesn't change as expected.

**Comment by blessanm86** [comment]
Try the fork from @xavivars. It has a couple of big fixes

**Comment by wardpeet** [comment]
Hey it does but you're pathPrefix is wrong in `gatsby-config.js`. The pathprefix needs to be the path where gatsby lives.

```
    "build:gatsby": "cross-env PUBLIC_URL=/web gatsby build --prefix-paths",
```

Please close when this solves your issue.

**Comment by isi-gach** [comment]
Hi @wardpeet 

I have upgraded plugin and made some changes in the sample repo to make more clear the issue
https://github.com/isi-gach/gastby-static

I have also created a video with the white page bug:
https://www.youtube.com/watch?v=0P_W3hnuTdw

I'm trying to serve the page from the following URL:
http://localhost:5000/public/index.html

It's good that now thanks to your plugin the URL don't change automatically to `http://localhost:5000/public/` but the page content flashes and a white page is displayed instead.

**Comment by isi-gach** [comment]
The issue is solved 0.2.0, thank you @wardpeet
